### ISSUE TITLE
Measure potential gain on UPDATE

### ISSUE THREAD

**Issue by sebastienros** [aspnet/benchmarks#469]
Can we apply this? https://github.com/TechEmpower/FrameworkBenchmarks/pull/3749/files

**Comment by benaadams** [comment]
Is this related to? https://twitter.com/TFBenchmarks/status/996834787150082049

**Comment by sebastienros** [comment]
I don't think so, what you are referring to is based on this issue: https://github.com/TechEmpower/FrameworkBenchmarks/issues/2204#issuecomment-241571077

Or you are right and then please correct me

**Comment by sebastienros** [comment]
@benaadams  There is a discussion going on on the google forum, title is "[frameworks] The Results Dashboard indicate and issue"

**Comment by benaadams** [comment]
Yeah, its what the tweet links to, with the comment:

> Postgres pipelining is in some ways analogous to automatic batching. We'd like more expert opinions on the following discussion:

**Comment by benaadams** [comment]
But people are making PRs for it... 

Would be worth (after measuring it does something); making a PR independent of other changes, stating what it is in title (as per linked PR) - then its ready or not pending decision?

**Comment by bhauer** [comment]
Based on prior concessions we made to the requirements, the *Updates* test type [permits batching of the UPDATE statements (but only the UPDATES)](https://www.techempower.com/benchmarks/#section=code).  Yes, you may use this in your implementation of the Updates test.

I believe recent conversations about the *Multiple-queries* test type (the conversation on the Google Group mailing list) reminded @michaelhixson that the Undertow implementation did not use a batching for updates despite that being permitted, hence the linked PR.

**Comment by benaadams** [comment]
Ah, makes sense - thank you for the clarification

**Comment by bhauer** [comment]
FWIW I refreshed my memory on that change to the Updates requirements, and putting aside any present-day regret I am feeling about permitting batching, I've now [also clarified the requirements to be more permissive on the use of transactions](https://twitter.com/BrianHauerTSO/status/997182287622979584).  Prior wording seemed inconsistent because batch updates were permitted, but transactions were only permitted in a more constrained manner.

**Comment by sebastienros** [comment]
Thanks @bhauer
